src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -49,7 +51,33 @@ Ready to get started? Make sure you have a [New Relic account](https://newrelic.
 
 Access to log management capabilities is limited by user type. See [user type capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type#user-type-capabilities).
 
-## Set up errors inbox [#get-started]
+## Use errors inbox directly from scoped views [#scoped]
+
+Scoped views allow you to view errors for entities in browser, mobile, and [Open Telemetry](https://docs.newrelic.com/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-introduction/#benefits). Access the errors inbox view for a specific entity by navigating to that entity in the browser or mobile monitoring view and selecting **Errors inbox** in the left nav. 
+
+<img
+  title="An image showing how to navigate to a scoped browser entity"
+  alt="This is an image of the main errors inbox UI inside a browser entity"
+  src={scopedBrowser}
+/>
+
+<figcaption>Traige errors for a browser or mobile entity by selecting **Errors inbox** in the left nav of the browser or mobile explorer.</figcaption>
+
+NOTE: Please replace image above with new browser image provided on Slack channel 
+
+
```

Comment:
It feels a bit redundant to me to have basically the same image twice.  Any reason not to add Otel (and APM??) to the list <figcaption> above?

Reply:
I agree. I'd rather use Otel here and then just list APM/Browser in  relevant section above:


Paragraph:
`Access the errors inbox view for a specific entity by navigating to that entity in the APM, browser, mobile, or OpenTelemetry monitoring view and selecting **errors inbox** in the left nav under **triage**`

Caption:
`Traige errors for an entity by selecting **errors inbox** in the left nav of the explorer.`

Reply:
Good point, sound good to me! Thanks Oscar.